tastytrade/account.py

```diff
@@ -174,23 +174,23 @@ class MarginReportEntry(TastytradeJsonDataclass):
     """
     description: str
     code: str
-    underlying_symbol: str
-    underlying_type: str
+    buying_power: Decimal
+    buying_power_effect: PriceEffect
     margin_calculation_type: str
     margin_requirement: Decimal
     margin_requirement_effect: PriceEffect
-    maintenance_requirement: Decimal
-    maintenance_requirement_effect: PriceEffect
-    buying_power: Decimal
-    buying_power_effect: PriceEffect
-    groups: List[Dict[str, Any]]
-    price_increase_percent: Decimal
-    price_decrease_percent: Decimal
+    point_of_no_return_percent: Decimal
```

Comment:
I believe this needs to be optional, I don't remember under what conditions it can be none but anything that's optional is optional for a reason